gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/bottomsheet/GravatarQuickEditorBottomSheet.kt

```diff
@@ -71,7 +71,7 @@ public fun GravatarQuickEditorBottomSheet(
 internal fun GravatarQuickEditorBottomSheet(
     gravatarQuickEditorParams: GravatarQuickEditorParams,
     authenticationMethod: AuthenticationMethod,
-    onAvatarSelected: (AvatarUpdateResult) -> Unit,
+    onAvatarSelected: () -> Unit,
```

Comment:
Shouldn't the `AvatarUpdateResult` be removed as well? And we need to update the docs for this callback in all the places mentioning about loading via hash with cache buster.

Reply:
Good catch! I've removed `AvatarUpdateResult` and added a small section about cache busting in the `Get Started`.